src/exit_codes.rs

```diff
@@ -1,4 +1,13 @@
-/// exit code 1 represents a general error
-pub const ERROR: i32 = 1;
-/// exit code 130 represents a process killed by signal SIGINT
-pub const SIGINT: i32 = 130;
+pub enum ExitCode {
```

Comment:
Maybe while we're at it, we could make the names a little bit more descriptive. Maybe `ExitCode::GeneralError` and `ExitCode::KilledBySigint`?